app/core/store/modules/me.js

```diff
@@ -10,6 +10,10 @@ export default {
   state: _.cloneDeep(emptyUser),
 
   getters: {
+    isInGodMode (state) {
+      return ((state != null ? state.permissions : undefined) || []).indexOf('godmode') > -1
```

Comment:
Is the intent here to use a less strict check so that the check `!= null` treats `undefined` and `null` the same?

Also interested in this syntax vs `((state || {}).permissions || [])`.

Also 95% of users support [Array.some](https://caniuse.com/#feat=mdn-javascript_builtins_array_some), so we could use that instead of `indexOf`. This has the best compatibility.

Reply:
Good out here I based this off of the other methods but it doesn't make sense.  Update this and the other methods in this block.  Regarding Array.some, are you saying that it has wider support than indexOf?

Reply:
No sorry. I re-read my confusing last sentence. I meant your current solution has best support. Sorry!